content/en/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm.md

```diff
@@ -529,7 +528,11 @@ kubectl --kubeconfig ./admin.conf proxy
 
 You can now access the API Server locally at `http://localhost:8001/api/v1`
 
-## Tear down {#tear-down}
+{{% /capture %}}
+
+{{% capture cleanup %}}
+
+### Tear down {#tear-down}
```

Comment:
looking at the preview i think there is a formatting error as this section is missing:
https://deploy-preview-18939--kubernetes-io-master-staging.netlify.com/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm/#tear-down

e.g.
https://deploy-preview-18939--kubernetes-io-master-staging.netlify.com/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm/#tear-down

perhaps a "capture" section is not closed somewhere?

Reply:
All captures are closed properly, it's just that the `task` template doesn't know about `{{% capture cleanup %}}`. You'll have to put this in `{{% capture steps %}}` i think.